### ISSUE TITLE
Service block out of alignment with DIDComm

### ISSUE THREAD

**Issue by brianorwhatever** [decentralized-identity/peer-did-method-spec#44]
The service property for didcomm is out of alignment with the current state of [didcomm](https://identity.foundation/didcomm-messaging/spec/#service-endpoint)

**Comment by dhh1128** [comment]
Agreed; there are some details to tweak. Updating the did method to realign it is easy. However, it may break existing implementations, so I'd like to know whether people using peer DIDs are reluctant to have the spec updated. Tagging @swcurran and @TelegramSam , who may have an opinion.

**Comment by brianorwhatever** [comment]
Yeah, it will definitely break existing implementations 😅
Wondering what the best path forward is as it's impossible to align with both this and didcommv2 at the moment. I may implement it this way to interoperate more broadly. maybe we should really consider changing the service block in didcomm back to how it was originally
